docs/en/features/census_configuration.md

```diff
@@ -1,155 +1,145 @@
 
-    Example:
-    ![Request Data - Path document type](../../img/remote_census/request-data-path-document-type-en.png)
-  - **Path for document number**: Path in the request structure that sends the Document Number.
+- **Path for document number**: Path in the request structure that sends the Document Number.
 
-    *NOTE: DO NOT FILL IN if the WebService does not require the Document Number to verify a user.*
+  *NOTE: DO NOT FILL IN if the WebService does not require the Document Number to verify a user.*
 
-    Example:
-    ![Request Data - Path document number](../../img/remote_census/request-data-path-document-number-en.png)
-  - **Path for date of birth**: Path in the request structure that sends the Date of Birth.
+  ![Request data - Path for document number is filled in with request.documento](../../img/remote_census/request-data-path-document-number-en.png)
+- **Path for date of birth**: Path in the request structure that sends the Date of Birth.
 
-    *NOTE: DO NOT FILL IN if the WebService does not require the Date of Birth to verify a user.*
+  *NOTE: DO NOT FILL IN if the WebService does not require the Date of Birth to verify a user.*
 
-    In the case of *Example* we will fill it in blank, since it is not necessary to send the date of birth to verify a user.
+  In *this example*, we will leave it blank, since it is not necessary to send the date of birth to verify a user.
 
-    Example:
-    ![Request Data - Path date of birth](../../img/remote_census/request-data-path-date-of-birth-en.png)
-  - **Path for Postal Code**: Path in the request structure that sends the Postal Code.
+  ![Request data - Path for date of birth is left blank](../../img/remote_census/request-data-path-date-of-birth-en.png)
+- **Path for Postal Code**: Path in the request structure that sends the Postal Code.
 
-    *NOTE: DO NOT FILL IN if the WebService does not require the Postal Code to verify a user.*
+  *NOTE: DO NOT FILL IN if the WebService does not require the Postal Code to verify a user.*
 
-    En el caso del *Example* lo dejaríamos en blanco, ya que no se necesita enviar el código postal para verificar a un usuario.
+  In *this example*, we will leave it blank, since it is not necessary to send the postal code to verify a user.
 
-    Example:
-    ![Request Data - Path postal code](../../img/remote_census/request-data-path-postal-code-en.png)
+  ![Request data - Path for postal code is left blank](../../img/remote_census/request-data-path-postal-code-en.png)
 
-1. **Response data**
+### Response data
 
-  In this section we will configure all the necessary fields to be able to receive the answer of the WebService and to verify a user in the application.
+In this section we will configure all the necessary fields to be able to receive the answer of the WebService and to verify a user in the application.
 
-  ![Response Data](../../img/remote_census/response-data-en.png)
+![Response data](../../img/remote_census/response-data-en.png)
```

Comment:
This image isn't present in the Spanish version :thinking:. Since we've removed the "request data" images, should we remove the "response data" images as well?